src/ILVerify/src/Program.cs

```diff
@@ -270,6 +278,28 @@ private void VerifyAssembly(PEReader peReader, EcmaModule module, string path)
                 methodCounter++;
             }
 
+            foreach (TypeDefinitionHandle typeHandle in metadataReader.TypeDefinitions)
+            {
+                // get fully qualified type name
+                var className = GetQualifiedClassName(metadataReader, typeHandle);
+                bool verifying = ShouldVerify(className);
+                if (_verbose)
+                {
+                    Write(verifying ? "Verifying " : "Skipping ");
+                    WriteLine(className);
+                }
+
+                if (verifying)
+                {
+                    var results = _verifier.VerifyInterface(peReader, typeHandle);
```

Comment:
the program [pass methods to verifier after filters](https://github.com/dotnet/corert/pull/6409/files#diff-6524fba58cf1727330522a5b324a780bL260) i we add also interface check we need to filter class name...my idea is to use Verify overload with TypeHandle for methods and also for interface check...to have a simple list of results. To do that i could check regex directly inside verifier...i don't know if this is a good idea(we need to move some code inside verifier), what do you think?

Reply:
I do not think the Verifier should have any built-in regex matching. The string matching based on the command line arguments should stay in the driver.

Reply:
Ok thank's!